view/frontend/web/js/view/payment/method-renderer/razorpay-method.js

```diff
@@ -413,7 +413,8 @@ define(
                         integration: 'magento',
                         integration_version: data.module_version,
                         integration_parent_version: data.maze_version,
-                    }
+                    },
+                    __referer : url.build('checkout/')
```

Comment:
@yashgit891 , We need this URL from the current headers, other wise this looks fine.